github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -93,3 +93,34 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-okd.sh"
+    - name: pull-kubernetes-nmstate-e2e-ocp
+      skip_branches:
+        - release-\d+\.\d+
```

Comment:
on https://github.com/kubevirt/project-infra/pull/295/files, indent is different, dont know if it matters
    skip_branches:
    - release-\d+\.\d+

Reply:
It does not matter.